app/src/main/res/layout/activity_entry_image.xml

```diff
@@ -60,7 +50,7 @@
             android:layout_height="wrap_content"
             android:layout_below="@id/entry_image_description_text_view"
             android:text="@string/upload_image_camera"
-            app:layout_widthPercent="50%"
+            app:layout_widthPercent="48%"
```

Comment:
I wanted some space between them, and I don't think `PercentRelativeLayout` allows me to set a margin... Is this a bad practice?